<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>email_input</class>
 <widget class="QMainWindow" name="email_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>696</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Signup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>confirm password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPassword"/>
      </item>
      <item>
       <widget class="QPushButton" name="signup_btn">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
background-color: white;
}
QLineEdit{
 background-color:#fa0000  ;
	border: none;
	border-bottom: 1px solid black;
	padding: 3px 5px;
}</string>
        </property>
        <property name="text">
         <string>signup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="login_btn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Already had account? Login here</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../assets/anh.jpg</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
